rules-tests/DowngradePhp80/Rector/FunctionLike/DowngradeStaticTypeDeclarationRector/config/configured_rule.php

```diff
@@ -1,11 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-use Rector\DowngradePhp80\Rector\FunctionLike\DowngradeStaticTypeDeclarationRector;
-use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
-
-return static function (ContainerConfigurator $containerConfigurator): void {
-    $services = $containerConfigurator->services();
-    $services->set(DowngradeStaticTypeDeclarationRector::class);
-};
```

Comment:
I'm curious, why is this config deleted? It's wierd test still passes without it, as it needs it to run :thinking:

Reply:
This file is orphan. I think it was copied [there](https://github.com/rectorphp/rector-src/blob/main/rules-tests/DowngradePhp80/Rector/ClassMethod/DowngradeStaticTypeDeclarationRector/config/configured_rule.php) instead of being moved .
Sorry I didn't mention that before.

Reply:
Ah, thanks for the info. It makes sense :+1: